internal/olm/manager.go

```diff
@@ -26,14 +26,16 @@ import (
 )
 
 const (
-	DefaultVersion = "latest"
-	DefaultTimeout = time.Minute * 2
+	DefaultVersion   = "latest"
+	DefaultTimeout   = time.Minute * 2
+	DefaultNamespace = "olm"
```

Comment:
should. not the `DefaultNamespace` be `olmresourceclient.OLMNamespace`?

Reply:
my thinking was to keep all  OLM related defaults in one place. So I moved `olmresourceclient.OLMNamespace` to this file 

https://github.com/operator-framework/operator-sdk/pull/2613/files#diff-d99b4452be120b4094387171182b4470L45

But I could be wrong about the intention of this manager. Let me know I can put this back in `olmresourceclient`

Reply:
I'm fine with this change.

Reply:
The problem is that we have Operator Namespace, Watch Namespaces, and then here the Namespace where OLM will be installed. So, I really think that is required we make clear that it is the OLM Namespace.

Reply:
I'l change Namespace to OLMNamespace at all places to avoid confusion